source/geod24/concurrency.d

```diff
@@ -1030,6 +1023,7 @@ protected:
 
         void notify () nothrow
         {
+            this.freezeTimer();
```

Comment:
Why do you call it `freeze` rather than `stop`? Do you ever resume that same timer? That's what it seems to imply.

Reply:
No particular reason and they dont ever get resumed. Stop would have been more appropriate. Will change.